README.md

```diff
@@ -72,21 +73,28 @@ cd front-end-monorepo
 docker-compose build
 ```
 
-`docker-compose up` runs local production builds of the project app at http://localhost:3000 and the content pages app at http://localhost:3001
+`docker-compose up` runs local production builds as follows:
+
+- content pages app at http://localhost:3001
+- project app at http://localhost:3000
+- root app at http://localhost:3002
```

Comment:
These ports could be revisited. It doesn't really matter as long as the configs are working, but it might make sense to have app-root as 3000, and adjust the others accordingly.

Reply:
I've changed the FEM ports to 3001, 3002, and 3003, just in case.